src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -457,7 +457,8 @@ static void create_stub_global_symbols(
             component,
             "java::" + id2string(parse_tree.parsed_class.name),
             symbol_table,
-            class_hierarchy);
+            class_hierarchy,
+            true);
```

Comment:
This is worth a comment here (include interfaces as interfaces can define fields) (of course in Java9, interfaces can define methods so I suppose we'll need to come back to this anyway...)

Reply:
Apparently default methods are actually in Java 8, but I guess this won't be the only problem with them!